mod/ckeditor/views/default/js/ckeditor/boot.js

```diff
@@ -0,0 +1,16 @@
+define(function (require) {
```

Comment:
Should this be `elgg/ckeditor/boot` since it is not written by the ckeditor team? Feels strange to squat on their namespace with our own code.